jupyter-extension/cypress/integration/repo-datum-mode-switch.ts

```diff
@@ -0,0 +1,60 @@
+describe('switching between repo and datum mode', () => {
+  beforeEach(() => {
+    cy.unmountAllRepos();
+    cy.resetApp();
+    cy.isAppReady();
+    cy.openMountPlugin();
+    cy.findAllByText('Mount');
+  });
+
+  it('should open datum mode', () => {
+    cy.findByTestId('Datum__mode').click();
+    cy.findAllByText('Test Datums').should('have.length', 1);
+    cy.findByTestId('Datum__inputSpecInput')
+      .invoke('attr', 'placeholder')
+      .should('contain', 'repo: images');
+  });
+
+  it('mounted repos appear in datum input spec and again when switching back', () => {
+    cy.findAllByText('Mount').first().click();
+    cy.findByTestId('ListItem__select').select('branch');
+    cy.findAllByText('Mount').first().click();
+    cy.findAllByText('Unmount').should('have.length', 2);
+
+    cy.findByTestId('Datum__mode').click();
+    cy.findByTestId('Datum__inputSpecInput')
+      .invoke('prop', 'value')
+      .should('contain', 'cross:');
+    cy.findByTestId('Datum__back').click();
+
+    cy.wait(3000);
```

Comment:
Looked at the test. Not sure why this is needed here. Our timeout should be more than enough in the settings file.